src/Dommel/Any.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Dapper;
+using System;
+using System.Data;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+
+namespace Dommel
+{
+    public static partial class DommelMapper
+    {
+        /// <summary>
+        /// Check if there's any entities on database.
+        /// </summary>
+        /// <typeparam name="TEntity">The type of the entity.</typeparam>
+        /// <param name="connection">The connection to the database. This can either be open or closed.</param>
+        /// <param name="transaction">Optional transaction for the command.</param>
+        /// <returns>True if there's at least one entity on database.</returns>
+        public static bool Any<TEntity>(this IDbConnection connection, IDbTransaction? transaction = null)
+        {
+            var sql = BuildAnyAllSql(GetSqlBuilder(connection), typeof(TEntity));
+            LogQuery<TEntity>(sql);
+            return connection.ExecuteScalar<bool>(sql, transaction);
+        }
+
+        /// <summary>
+        /// Check if there's any entities on database.
+        /// </summary>
+        /// <typeparam name="TEntity">The type of the entity.</typeparam>
+        /// <param name="connection">The connection to the database. This can either be open or closed.</param>
+        /// <param name="transaction">Optional transaction for the command.</param>
+        /// <returns>True if there's at least one entity on database.</returns>
+        public static Task<bool> AnyAsync<TEntity>(this IDbConnection connection, IDbTransaction? transaction = null)
+        {
+            var sql = BuildAnyAllSql(GetSqlBuilder(connection), typeof(TEntity));
+            LogQuery<TEntity>(sql);
+            return connection.ExecuteScalarAsync<bool>(sql, transaction);
+        }
+
+        /// <summary>
+        /// Check if there's any entities matching the specified predicate.
+        /// </summary>
+        /// <typeparam name="TEntity">The type of the entity.</typeparam>
+        /// <param name="connection">The connection to the database. This can either be open or closed.</param>
+        /// <param name="predicate">A predicate to filter the results.</param>
+        /// <param name="transaction">Optional transaction for the command.</param>
+        /// <returns>True if there's at least one entity that match the specified predicate.</returns>
```

Comment:
```suggestion
        /// <returns><c>true</c> if there's at least one entity in the database that matches the specified predicate; otherwise, <c>false</c>.</returns>
```